### ISSUE TITLE
dockerfile stopped working

### ISSUE THREAD

**Issue by zubair1502** [peterx7803/PointNet-in-Docker#1]
Hello @peterx7803 , Thanks to share this dockerfile. I am new to Docker and I want to use this dockerfile to run pointnet. When I am trying to build an image from this file, it got stuck after asking my geographical area. can you please suggest me how can I solve this issue? 
Thanks!

**Comment by javierBarandiaran** [comment]
I have replaced this
FROM tensorflow/tensorflow:latest-gpu
by this
FROM tensorflow/tensorflow:1.0.1-gpu
ARG DEBIAN_FRONTEND=noninteractive

**Comment by zubair1502** [comment]
Hi @javierBarandiaran 
Thanks for the suggestion. I think you are right because when I tried without using docker, I faced the tensorflow version issues but there I was able to crack it. 
At that time I had no idea that this problem is occuring because of tensorflow version because it was not showing any error related to tensorflow. 
I will try it again by keeping your suggestion in mind. 

Cheers

**Comment by javierBarandiaran** [comment]
Yes, I also changed the tensorflow version because part_seg/train.py failed. To solve the problem building the docker image you just need to add ARG DEBIAN_FRONTEND=noninteractive
